lib/util/http/github.ts

```diff
@@ -140,6 +140,22 @@ interface GraphqlOptions {
   fromEnd?: boolean;
 }
 
+function constructAcceptString(input?: any): string {
+  const defaultAccept = 'application/vnd.github.v3+json';
+  const appModeAccept = 'application/vnd.github.machine-man-preview+json';
+  const acceptStrings = typeof input === 'string' ? input.split(/\s*,\s*/) : [];
```

Comment:
Why do we need the type check? Shouldn't it always be a string? Or `null` or `undefined`?

Reply:
Don't like this too, but generic http header can be also `string[]`or `number`